### ISSUE TITLE
Recompiling with a different line count.

### ISSUE THREAD

**Issue by blieque** [emreyolcu/discrete-scroll#2]
Firstly, sorry to submit an issue, but I wasn't sure how else to contact you (I don't have enough rep on AskDifferent to comment).

I'm not overly familiar with macOS and don't have Xcode installed. Is there a way to recompile the app with 1 or 2 lines (or ~1.5; are decimals possible?) of scrolling? My mouse has a particularly fast scroll wheel, and 3 is a bit quick. If not, could you upload a binary with slower scrolling or some configuration mechanism.

Thank-you for your work, mind! As quick as it is, this scrolling is heaps better than the OS's.

**Comment by emreyolcu** [comment]
Hi,

I uploaded two new binaries as part of [release v0.1.1](https://github.com/yolcu/discrete-scroll/releases/tag/v0.1.1). 

As far as I know, the CoreGraphics API does not expose a way to scroll for a fraction of a line and when I set `LINES` to a decimal value it seems to get truncated to an integer.

I intend to one day write a preference pane out of this program with configuration at runtime but currently I do not have enough time to learn more Cocoa programming.
